coralproject/talk
# Add stable classnames to more DOM elements in stream

## What does this PR do?

Adds more stable classnames to stream DOM elements to prevent usage of attribute selectors in custom stylesheets, making the DOM more future-proof and making customization simpler and more reliable. I tried to be conservative and base my additions on the custom stylesheets we maintain at Vox Media. The one exception is the addition of the `coral-icon` class to all icons. This somewhat contradicts vinh's initial direction of adding stable classnames to specific instances rather than components, which i assume was to encourage authors of custom stylesheets to use css variables where possible. I've made this exception for icons because they account for a large number of the attribute selectors i've seen at Vox Media and in the wild, as changing icons is a very common customization. In combination with the container classes already in place and that have been added here, this one classname should provide enough specificity to enable simple customization. 

## These changes will impact:


## How do I test this PR?

View the stream and inspect impacted elements to see new classnames. Inspect the DOM in the "preferences" tab to see new classnames for all headings and sections.
